cfme/tests/webui/test_expression_editor.py

```diff
@@ -0,0 +1,44 @@
+import fauxfactory
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+
+class TestDeleteExpressions(object):
```

Comment:
are you sure you need this class at all ?

Reply:
It seems that after lots of simplifications now it isn't really necessary :)